gp-includes/things/translation-set.php

```diff
@@ -142,7 +142,7 @@ public function normalize_fields( $args ) {
 		}
 
 		if ( ! empty( $args['slug'] ) ) {
-			$args['slug'] = sanitize_title( $args['slug'] );
+			$args['slug'] = gp_sanitize_project_name( $args['slug'] );
```

Comment:
Looks like we should find another name for the function. Maybe `gp_sanitize_for_url()` wasn't that bad...